cmd/argocd/commands/logout.go

```diff
@@ -10,6 +10,23 @@ import (
 	argocdclient "github.com/argoproj/argo-cd/v2/pkg/apiclient"
 	"github.com/argoproj/argo-cd/v2/util/errors"
 	"github.com/argoproj/argo-cd/v2/util/localconfig"
+	"github.com/argoproj/argo-cd/v2/util/templates"
+)
+
+var (
+	logoutExample = templates.Examples(`
```

Comment:
You'll need to set the `Example: logoutExample` field on the target `Command` instantiation.

No worries about the tests, the CI will ensure that tests pass. :-)